data/src/main/java/com/expediagroup/streamplatform/streamregistry/data/Status.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.data;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Lob;
+
+import lombok.Data;
+
+@Deprecated // do we need this since it holds one field?
```

Comment:
We're going to add support for multiple agents to save status, basically a map of status, keyed by name.

Please can you remove the @Deprecated annotation

Reply:
done